Homework2/MatrixTask/ElementChangedEventArgs.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MatrixTask
+{
+    public class ElementChangedEventArgs<T>: EventArgs
+    {
+        public object OldElement { get; set; }
```

Comment:
```suggestion
        public T OldElement { get; set; }
```

current generic type T is not using at all

Reply:
done, thank you